packages/neutron-vesting-lp-vault/src/error.rs

```diff
@@ -0,0 +1,34 @@
+use cosmwasm_std::{ConversionOverflowError, StdError};
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("{0}")]
+    ConversionOverflowError(#[from] ConversionOverflowError),
```

Comment:
Can't see usages for that one (both here and at the lockdrop vault where it's added as well)

Reply:
What about `contracts/dao/voting/vesting-lp-vault/src/contract.rs:258`? It is used there implicitly.

Reply:
Let's maybe do
```rust
    Ok(VotingPowerAtHeightResponse {
        power: get_voting_power(deps, &config, height, &query_msg)?
            .numerator()
            .try_into().map_err(|e| StdError::from(e))?,
        height,
    })
```
instead of
```rust
    Ok(VotingPowerAtHeightResponse {
        power: get_voting_power(deps, &config, height, &query_msg)?
            .numerator()
            .try_into()?,
        height,
    })
```
?
It will allow us to avoid adding new `ConversionOverflowError` error which is already a part of added StdError

Reply:
sounds reasonable